packages/block-editor/src/hooks/style.js

```diff
@@ -269,28 +269,28 @@ export function omitStyle( style, paths, preserveReference = false ) {
 /**
  * Override props assigned to save component to inject the CSS variables definition.
  *
- * @param {Object}                    props      Additional props applied to save element.
- * @param {Object}                    blockType  Block type.
- * @param {Object}                    attributes Block attributes.
- * @param {?Record<string, string[]>} skipPaths  An object of keys and paths to skip serialization.
+ * @param {Object}                    props           Additional props applied to save element.
+ * @param {Object|string}             blockNameOrType Block type.
+ * @param {Object}                    attributes      Block attributes.
+ * @param {?Record<string, string[]>} skipPaths       An object of keys and paths to skip serialization.
  *
  * @return {Object} Filtered props applied to save element.
  */
 export function addSaveProps(
 	props,
-	blockType,
+	blockNameOrType,
 	attributes,
 	skipPaths = skipSerializationPathsSave
 ) {
-	if ( ! hasStyleSupport( blockType ) ) {
+	if ( ! hasStyleSupport( blockNameOrType ) ) {
 		return props;
 	}
 
 	let { style } = attributes;
 	Object.entries( skipPaths ).forEach( ( [ indicator, path ] ) => {
 		const skipSerialization =
 			skipSerializationPathsSaveChecks[ indicator ] ||
```

Comment:
Quick question for @andrewserong: how should the background image styles be applied? They seem to be skipped here.

Or should we be also using `addSaveProps` in hooks/background.js? Trying to work out why the CSS isn't being applied to the group container 🤔

Reply:
Good catch! Hrm, at first glance, it looks like this PR likely breaks it. The idea was for `addEditProps` to be adding the inline style in the edit context, but not serialize it on save.

We'll likely soon have other background image properties that need to be output in the editor view but not serialized to post content.

> Or should we be also using addSaveProps in hooks/background.js? 

I think that should work! I think we'd just call `addSaveProps` in `hooks/background.js` with a `skipPaths` param that allows the background image styles to be output in that context. What do you think?

Reply:
I'll have a look at it, thanks for testing. Would be great if at some point we add e2e tests.

Reply:
I found the issue: I used `skipSerializationPathsSave` instead of `skipSerializationPathsEdit`. Fixed it in bf0950ddc53be47f18cacb7de2fd8354b14d11c3.

Reply:
Oh, well spotted! LGTM now